1.java

```diff
@@ -0,0 +1,24 @@
+@FunctionalInterface
+interface InterestCalculator{
+    int calculate(int p,int r,int t);
+}
+@FunctionalInterface
```

Comment:
Add Comments in the code.

Reply:
the return type of calculate could have been float, because we are printing the float value at the end.